src/app/game-state/followers.service.ts

```diff
@@ -75,10 +75,14 @@ export class FollowersService {
         if (this.characterService.characterState.equipment.rightHand && 
           this.characterService.characterState.equipment.rightHand.weaponStats){
           this.characterService.characterState.equipment.rightHand.weaponStats.durability += follower.power;
+          this.characterService.characterState.equipment.rightHand.weaponStats.baseDamage += Math.floor(follower.power/10);
```

Comment:
This new damage (and value) added by followers should make it possible to improve weapons fast enough to win against Death.